src/i18n.js

```diff
@@ -0,0 +1,55 @@
+import i18n from "i18next";
+import Backend from "i18next-xhr-backend";
+import LanguageDetector from "i18next-browser-languagedetector";
+import { initReactI18next } from "react-i18next";
+
+i18n
+  .use(Backend)
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    fallbackLng: 'en',    
+    languages: ['sv', 'en'],
+    resources: {
+      en: {
+        navbar: {
+          home: 'Home',
+          language: 'Languages',
+          swedish: 'Swedish',
+          english: 'English',
+          categories: 'Categories',
+          environment: 'Environment',
+          medicine: 'Medicine',
+          outreach: 'Outreach',
+          search: 'Search...',
+          login: 'Log in',
+          signup: 'Sign up',
+          create_article: 'Create Article',
+          subscribe: 'Subscribe'
+        }
+      },
```

Comment:
In what way is reading translations from a json file buggy?

Reply:
We followed the demo from Faraz word for word, but the translation.json files would not import with the same configuration in our app.  Manual imports of the files also failed.  The format you see above was the best functional compromise we could find and the documentation is thin in troubleshooting this particular issue.

Reply:
This is how the app looks when trying to use locales folders.

https://recordit.co/C6lxr8feOt

Reply:
The recorded gif does not tell me anything. What am I looking at?

Reply:
Well, word-for-word could be debated. Please see this commit and make the changes or cherry-pick it into your branch for proper developer attribution https://github.com/tochman/gnosis-client/commit/05b37d295bc55da9ed381bdf00a58e7c5c12e325